yksgi/src/lib.rs

```diff
@@ -109,32 +121,53 @@ impl SGInterp {
     /// Create a new stopgap interpreter and initialise it to start interpretation at the location
     /// given by a basic block index, instruction index, and function name.
     /// FIXME: Support initialisation of multiple frames.
-    pub unsafe fn new(bbidx: usize, instridx: usize, fname: &CStr) -> SGInterp {
+    pub unsafe fn new(activeframes: &[FrameInfo]) -> SGInterp {
         // Get AOT module IR and parse it.
         let module = Module::from_bc();
+
+        // Initialise frames.
+        let mut frames = Vec::new();
```

Comment:
Maybe `Vec::with_capacity(activeframes.len())` to avoid `realloc`s?

Reply:
Fixed in b57d92c1d9.